test/org/opensolaris/opengrok/web/ProjectHelperTestBase.java

```diff
@@ -252,23 +256,12 @@ public static void tearDownClass() {
     }
 
     protected void invokeRemoveAll() {
-        try {
-            Method method = AuthorizationFramework.class.getDeclaredMethod("removeAll");
-            method.setAccessible(true);
-            method.invoke(AuthorizationFramework.getInstance());
-        } catch (Exception ex) {
-            Assert.fail("invokeRemoveAll should not throw an exception");
-        }
+        env.setPluginStack(new AuthorizationStack(AuthControlFlag.REQUIRED, "default-stack"));
```

Comment:
This is necessary because the application changes the stack at runtime directly into `env` and does not have its local copy in the auth. framework.